helm/charts/aleph/README.md

```diff
@@ -11,7 +11,7 @@ Helm chart for Aleph
 | global.amazon | bool | `true` | Are we using AWS services like s3? |
 | global.google | bool | `false` | Are we using GCE services like storage, vision api? |
 | global.image.repository | string | `"alephdata/aleph"` | Aleph docker image repo |
-| global.image.tag | string | `"3.13.1-rc3"` | Aleph docker image tag |
+| global.image.tag | string | `"3.13.1-rc6"` | Aleph docker image tag |
```

Comment:
This is the only thing that doesn't look right, but perhaps this was broken for a while?

Reply:
Yeah, should be 3.13.2-rc3. Problem was my over zealous prettier plugin reformatting this documents causing bump2version to stop working in this file because it uses a simple find/replace